django_pin_auth/auth_backend.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth import get_user_model
+
+from .models import SingleUseToken
+
+class PinBackend(object):
```

Comment:
Auth backend should also implement a [`get_user`](https://docs.djangoproject.com/en/1.11/topics/auth/customizing/#writing-an-authentication-backend) method

Reply:
See a4a5ca393ffc553202b266bdea790768a119f8f8